src/Structures/Base.js

```diff
@@ -1,6 +1,8 @@
+import _ 	            from 'lodash'
+import Vue              from "vue";
+
 import Request          from '../HTTP/Request.js'
-import ValidationError  from '../Errors/ValidationError.js'
-import {autobind }       from '../utils.js';
+import {autobind }      from '../utils.js';
```

Comment:
How could this have worked before if it wasn't imported? Needs a space after the `{` to be consistent also.

Maybe a style guide / linter is a good idea. 🤔